### PR TITLE
Controlled exception on find_latest_package (Github Driver)

### PR DESCRIPTION
**Pull request by playmono**

### CODE REVIEW

File: src/Package_Command.php [hunk 1]
```diff
@@ -676,10 +676,14 @@ private function show_packages( $context, $packages, $assoc_args ) {
...
-						$update = 'available';
-						$update_version = $latest->getPrettyVersion();
+					try {
+						$latest = $this->find_latest_package( $package, $this->get_composer(), null );
+						if ( $latest && $latest->getFullPrettyVersion() !== $package->getFullPrettyVersion() ) {
+							$update = 'available';
+							$update_version = $latest->getPrettyVersion();
+						}
+					} catch (Exception $e) {
+						$update = $update_version = 'Error whith Github Driver';
```
Comment (gitlost) [inline src/Package_Command.php]: It'd be better I think to just use a simple `"error"` message in the table and warn giving the full exception message eg
```
$update = $update_version = 'error';
WP_CLI::warning( $e->getMessage() );
```

Comment (playmono) [reply src/Package_Command.php]: Yes, I remember being prompted for my ssh password but I don't remember in which case, sorry.

Comment (gitlost) [reply src/Package_Command.php]: Okay ta, I think to get around possible issues like that a new `--non-interactive` option should be added which I'll do in a separate issue/PR. For now I'm just going push my suggested change to your PR and commit if that's alright...

**Comment by gitlost** [comment]
Sigh keep getting either "packagist.org" as here now or "getcomposer.org" (https://github.com/wp-cli/entity-command/pull/122#issuecomment-354881484) connection timeouts on Travis lately... restarting job...

**Pull request merged**